acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -118,7 +101,6 @@ def stop_firewall_on(host)
 
       def install_repos_on(host, project, sha, repo_configs_dir)
```

Comment:
`repo_configs_dir` is no longer used, and could be removed (and all the call sites updated)